app/src/auth/utils/getSessionToken.js

```diff
@@ -2,9 +2,22 @@
 
 import StreamrClient from 'streamr-client'
 
-export default async (auth: Object): Promise<?string> => (
-    new StreamrClient({
-        restUrl: process.env.STREAMR_API_URL,
-        auth,
-    }).session.getSessionToken()
-)
+const parseError = async (error) => {
+    try {
+        return new Error(JSON.parse(error.body).message)
+    } catch (e) {
+        // Noop. Will rethrow.
```

Comment:
Perhaps worth a comment like "if there was an error parsing the error.body, just use the original error", took me a few reads to follow what was going on. Maybe save a line and return in the catch directly?

Reply:
Will do that. Sorry. ;)